app/views/shared/_sharing_widget.html.erb

```diff
@@ -1,2 +1,51 @@
-<h2>Spread the word</h2>
-<p>You give strangers a good rep. You’re the type that asks “How are you?” and means it. You opt for high fives over handshakes. And the friends you attract are similarly awesome. They’ll thank you when you tell them about TWS.</p>
\ No newline at end of file
+<div class="sharing-info">
+  <h2>Spread the word</h2>
+  <p>You give strangers a good rep. You’re the type that asks “How are you?” and means it. You opt for high fives over handshakes. And the friends you attract are similarly awesome. They’ll thank you when you tell them about TWS.</p>
+</div>
+
+<div class="sharing-box">
+
+  <div class="sharing-preview sharing-preview-bounce" id="sharing-preview">
+    <%= image_tag 'landing/landing-graphic-redesign.jpg', id: 'sharing-preview-image' %>
+    <div class="sharing-preview-text"><span id="sharing-preview-text-content">Tea With Strangers is all about making meaningful interactions more fun.</span> // <a href="http://teawithstrangers.com/">teawithstrangers.com</a></div>
+  </div>
```

Comment:
TODO: @ankitshah811 Decide on new image and set that as the `og:image` in the header as well as the default image  here.